### ISSUE TITLE
Rating (Form) - Ensure color is not the sole means of communicating information - (2036570496)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5630]
### Module: 
 40 Form 

### Violation: 
 Ensure color is not the sole means of communicating information 

### WCAG Reference: 
- [1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html) 

### Instance ID: 
 2036570496 

### Severity: 
 10 

### Description: 
 [Issue]
There is image after the "How many stars would you give Calcite components?" that uses color alone to visually indicate selected state information.

[User Impact]
Users with color blindness will have difficulty identifying the colors used to convey this information.

[Code Reference] 
 ```
<label class="star selected" for="calcite-ratings-29af6f19-da8e-6288-620e-bc02ffec3557-1">
  (...)
</label>

.hovered, .selected, :host([read-only]) .average, :host([read-only]) .fraction {
    color: var(--calcite-ui-brand);
}
 ``` 

### Note: 
 [Suggestion]
Ensure color is not the sole means of communicating information. Color can be used as long as other visual indicators of the information are also present. Common additional visual indicators include underlining, bolding, additional on-screen text conveying the information, or change in shape or size.
 

### Media Type: 
 Color and Contrast 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
![image](https://user-images.githubusercontent.com/5023024/202555056-0d3fc0e6-25d2-4fcd-b5bd-8243368f6c50.png)

**Comment by geospatialem** [comment]
This doesn't seem like a valid issue as selections are filled and colored, whereas unselected values are not filled or colored. Closing and marking as resolved.
